freqtrade/exchange/binance.py

```diff
@@ -217,3 +224,58 @@ def load_leverage_tiers(self) -> Dict[str, List[Dict]]:
                     raise OperationalException(e) from e
         else:
             return {}
+
+
+
+    @retrier
+    @cached(cache=TTLCache(maxsize=100, ttl=10), lock=Lock())
+    def get_deslist_schedule(self):
+        try:
+
+            delist_schedule = self._api.sapi_get_spot_delist_schedule()
```

Comment:
considering that this is a private (direct) endpoint - i think we should have a form to unify it. Usually, that's taken care of by ccxt, but in this case, that unification / abstraction isn't available (yet?).


i know that for example bybit has a similar endpoint (we need to parse the text to make that work) - but i think what the exchange returns should be unified across exchanges (across the exchanges that support it, anyway) - so it's easier to use from a consumer side (not necessarily the strategy - anything outside of the exchange section should be seen as a consumer of this function).